tests/lib/prepare.sh

```diff
@@ -1171,6 +1171,11 @@ setup_reflash_magic() {
     # remove the above debug lines once the mentioned bug is fixed
     snap install "--channel=${CORE_CHANNEL}" "$core_name"
     UNPACK_DIR="/tmp/$core_name-snap"
+    # make sure that the unpack directory does not exist, which may happen a
+    # previous attempt of 'core' suite prepare failed on this host (followed by
+    # successful restore), and we still want to give this one a chance so that
+    # we're not wasting CPU cycles
+    rm -rf "$UNPACK_DIR"
```

Comment:
Is there a reason we don't do `mktemp -d` instead, either in the core suite prepare or here? Must it be a meaningful name? 

I'd rather we not waste additional IO cycles on the already slow VMs removing files if we can just let them be

Reply:
Updated to create a temp directory, but left a comment that we have no reasonable way of cleaning it up right now (or at least not without complicating this further)

Reply:
Do we still need the `rm -rf`?